docs/api/extras/geometries/TextGeometry.html

```diff
@@ -20,7 +20,7 @@
 		<h3>[name]([page:String text], [page:Object parameters])</h3>
 		<div>
 		text — The text that needs to be shown. <br />
-		parameters — Object that can contain the following parameters.
+		parameters — Object that can contains the following parameters.
```

Comment:
This line 23 was correct to begin with.  Should have a semicolon if it will refer to a list.